crates/cloudproof/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! Meta crate in order to merge other crates
+
+#[allow(unused_imports)]
```

Comment:
Maybe `pub use` to avoid needing this macro?

Reply:
thanks